### DISCUSSION TITLE
Hide reviewers approval from other reviewers

### DISCUSSION THREAD

**Discussion by Stocco** [community/community#10167] [Pull Requests]
The title sounds crazy, but It would be really nice if there was an option to hide approvals from other reviewers in a PR. Sometimes people tend to do less critical reviews on the code if someone you trust already did the review on the code and this can lead into bugs that everyone reviewed but no one saw it.
I know that it should be a cultural thing but it is in our subconscious for many of my partners during the code review process. The moment you see other approvals you feel "safe" which you shouldn't be.
What do you guys think?

**Comment by Gil82OM** [comment]
![chrome_qrcode_1640867575288](https://user-images.githubusercontent.com/94519011/149623702-befacebd-9842-41ae-b4da-135b0d29cc41.png)

**Reply by Gil82OM** [reply]
> ![chrome_qrcode_1640867575288](https://user-images.githubusercontent.com/94519011/149623702-befacebd-9842-41ae-b4da-135b0d29cc41.png)

**Comment by Gil82OM** [comment]
(https://user-images.githubusercontent.com/94519011/149623761-5261f77e-6643-449c-9367-75227368085c.jpghttps://user-images.githubusercontent.com/94519011/149623761-5261f77e-6643-449c-9367-75227368085c.jpghttps://www.facebooksoniasolis.com/profile.php?id=100034373258234)

**Comment by Gil82OM** [comment]
I'm really starting and I would like to learn would you help me?
